examples/guestbook/frontend-service.json

```diff
@@ -17,6 +17,7 @@
       ],
       "selector":{
          "name":"frontend"
-      }
+      },
+      "createExternalLoadBalancer": true
```

Comment:
This may make the guestbook e2e test more flaky, since many of the tests that create external load balancers appear to be flakier than those that don't. 

@quinton-hoole